askup/models.py

```diff
@@ -372,8 +374,12 @@ def vote(self, user_id, value):
             question_id=self.id,
             voter_id=user_id,
         )
-        vote_value = self.vote_value + value
-        return vote_value, 'Thank you for your vote!'
+        self.vote_value = F('vote_value') + value
```

Comment:
I have an suspicion that you vote twice. In Vote.save and here. But finally there is a correct result.